### DISCUSSION TITLE
Show More" Button for Long Text in Memo

### DISCUSSION THREAD

**Discussion by ramzsey** [usememos/memos#4134] [Q&A] [answered]
Thank you for this amazing tool! I was wondering if it would be possible to add a "show more" button for long text in memos. For example:

When creating a memo with a long text, it would be great if the memo list could initially display a shortened version of the text, along with a "show more" option to reveal the full content.

**Comment by ramzsey** [comment]
I have just found out that it does exist, but how it detects the text and how you can specify that, no idea whatsoever )
I probably need a help with that now )
![Screenshot(1)](https://github.com/user-attachments/assets/c55621dc-fb8e-45a4-bb2e-91a7d5e289bf)

**Comment by RoccoSmit** [comment, marked as answer]
`settings` -> `memo` -> `Enable auto compact`

**Reply by ramzsey** [reply]
> `settings` -> `memo` -> `Enable auto compact`

Thank you!
